internal/extproc/translator/openai_awsbedrock.go

```diff
@@ -128,14 +146,24 @@ func (o *openAIToAWSBedrockTranslatorV1ChatCompletion) openAIToolsToBedrockToolC
 	if openAIReq.ToolChoice != nil {
 		switch reflect.TypeOf(openAIReq.ToolChoice).Kind() {
 		case reflect.String:
-			if openAIReq.ToolChoice.(string) == "auto" {
+			toolChoice := openAIReq.ToolChoice.(string)
```

Comment:
are you 100% sure this will be a string? then why the toolchoice is interface in the first place?

Reply:
ok the reflection above.... that is totally overkill and unnecessary